Explorer/Assets/DCL/Rendering/Reflections/ReflectionProbeRenderer.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class ReflectionProbeRenderer : MonoBehaviour
+{
+    public float IntervalInSeconds = 5f;
+    private ReflectionProbe reflectionProbe;
+    private float timer;
+
+    private void Start()
+    {
+        RenderSettings.defaultReflectionMode = UnityEngine.Rendering.DefaultReflectionMode.Custom;
+        reflectionProbe = GetComponent<ReflectionProbe>();
+        timer = IntervalInSeconds;
+    }
+
+    private void Update()
+    {
+        timer += Time.deltaTime;
+
+        // Update the reflection probe after timer
+        if (timer >= IntervalInSeconds)
+        {
+            _ = reflectionProbe.RenderProbe();
```

Comment:
We could also try setting up the `timeSlicingMode` of `reflectionProbe` to see if we can reduce the performance impact even more, which also implies waiting for the render to end with `reflectionProbe.IsFinishedRendering(int id)`

Based on the documentation:
- `AllFacesAtOnce` takes 8 frames to finish
- `IndividualFaces` takes 14 frames to finish

Since the skybox updates slowly, it's not going to be noticeable. 

What do you think?

Reply:
I've added the timeslicing mode inside of the interval. So it nows only render after x seconds, waits the render to be finished after 14 frames, and replaces environment reflection texture and resets.